charts/cloudzero-agent/CHANGELOG.md

```diff
@@ -9,6 +9,68 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Resource consumption is reduced by limiting ingestion of unneeded metrics and metric labels
+- Moved validator to [dedicated repository](https://github.com/Cloudzero/cloudzero-agent-validator)
+- Added validator as pod lifecycle hook (PostStart, PreStop) to track pod up/down transitions used to monitor data loss
+
+## [0.0.9]
+
+### Added
+- `prometheusConfig.scrapeJobs` is added to allow more granular control of scrape jobs.
+- Now `region` is a required input parameter
+- Added environment validation check to ensure request authorization / API key is set up
+
+### Fixed
+- `prometheusConfig.configOverride` only replaces the scrape_configs in the Prometheus configuration.
+
+## [0.0.8]
```

Comment:
I think you have all of these lines duplicated

Reply:
fixed!